src/main/java/com/example/GymProject/dao/TrainerDao.java

```diff
@@ -114,7 +93,7 @@ public void init() throws Exception {
                 trainerMap.put(userId, trainer);
             }
         } catch (Exception e) {
-            logger.warning("File can not be found!");
+            logger.error("File can not be found!");
             throw new FileNotFoundException("Wrong File");
```

Comment:
I'm ending with this exception right now.  Cannot start the app. Please use `Resource` instead of `File` on line 73 as it was previously mentioned.